en/docs/administer/changing-to-ibm-db2.md

```diff
@@ -90,180 +161,116 @@ Follow the steps below to change the type of the default datasource.
+       ```
+
+  3.  Navigate to the
+        `               <IS_HOME>/repository/conf/deployment.toml              `
+        file.
+  4.  Disable the `               defaultAutoCommit              `
+        by defining it as `false`.
+  5.  Add the `                commitOnReturn               `
+        property and set it to true.
+                         
+    - `WSO2_IDENTITY_DB` `deployment.toml` Configurations.
+        
+       ``` toml
+       [database.identity_db.pool_options]
+        defaultAutoCommit="false"
+        commitOnReturn="true"
+       ```
+       
+    - `WSO2_SHARED_DB` `deployment.toml` Configurations.
+            
+       ``` toml
+       [database.shared_db.pool_options]
+        defaultAutoCommit="false"
+        commitOnReturn="true"
+       ```    
+            
+### Configure the connection pool to rollback pending transactions on connection return
+
+  1.  Navigate to the
+        `<IS_HOME>/repository/conf/deployment.toml`            `
+        file.
+  2.  Disable the
+        `                defaultAutoCommit               ` by
+        defining it as `false`.
+
+  3.  Set the `                rollbackOnReturn               `
+        property to the datasources as true.
+
+
+    - `WSO2_IDENTITY_DB` `deployment.toml` Configurations.
+        
+       ``` toml
+       [database.identity_db.pool_options]
+        defaultAutoCommit="false"
+        rollbackOnReturn="true"
+       ```
+       
+    - `WSO2_SHARED_DB` `deployment.toml` Configurations.
+            
+       ``` toml
+       [database.shared_db.pool_options]
+        defaultAutoCommit="false"
+        rollbackOnReturn="true"
+       ```
+
+The elements in the above configuration are described below:
+
+ | **Element**          | **Description**                                                                                                                                                                                                                                                                                                                                                                            |
+ |----------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+ | **commitOnReturn**   | If `                defaultAutoCommit               ` =false, then you can set `                commitOnReturn               ` =true, so that the pool can complete the transaction by calling the commit on the connection as it is returned to the pool. However, If `                rollbackOnReturn               ` =true then this attribute is ignored. The default value is false. |
```

Comment:
However, if the `                rollbackOnReturn               ` property is set to  `true`,

Reply:
done